proto/file/file.proto

```diff
@@ -67,6 +68,15 @@ message GetUploadByIDResponse {
   string fileID = 6;
 }
 
+message DeleteUploadByKeyRequest {
```

Comment:
It's not just by key, but by key and bucket.
 I'd just call it `DeleteUploadRequest`

Reply:
in update.repository.ts have method updateByKey and she also uses with key and bucket